met-api/src/met_api/models/comment.py

```diff
@@ -112,12 +129,12 @@ def add_all_comments(cls, comments: list, session=None) -> DefaultMethodResult:
         return DefaultMethodResult(True, 'Comments Added', 1)
 
     @classmethod
-    def update_comment_status(cls, comment_id, status_id, reviewed_by) -> DefaultMethodResult:
+    def update_comment_status(cls, submission_id, status_id, reviewed_by) -> DefaultMethodResult:
```

Comment:
I think this is an existing code , but feel free to return CommentModel | None and remove the defaultmethodresult